roles/shiftstack/tasks/test_config.yml

```diff
@@ -53,7 +54,7 @@
         KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
       ansible.builtin.command:
         cmd: >
-          oc rsync -n {{ cifmw_shiftstack_client_pod_namespace }}
+          oc rsync -n {{ cifmw_shiftstack_client_pod_namespace }} --exclude={{ cifmw_shiftstack_exclude_artifacts_regex }}
```

Comment:
Does it make sense to have this new parameter as optional? If so, I think we should render it only if not empty.

Reply:
Addressing your comment. Thanks for the input!